test/framework/src/main/java/org/elasticsearch/test/AbstractXContentTestCase.java

```diff
@@ -54,21 +54,21 @@
         for (int runs = 0; runs < numberOfTestRuns; runs++) {
             T testInstance = instanceSupplier.get();
             XContentType xContentType = randomFrom(XContentType.values());
-            BytesReference shuffled = toShuffledXContent(testInstance, xContentType, toXContentParams,false,
-                createParserFunction, shuffleFieldsExceptions);
+            BytesReference xContent = XContentHelper.toXContent(testInstance, xContentType, ToXContent.EMPTY_PARAMS, false);
```

Comment:
The error mentioned below is most likely cause by not using the "toXContentParams" here as before mit using the EMPTY_PARAMS. I think this needs to be changed.